backend/app/views/spree/admin/images/_image_row.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr id="<%= spree_dom_id image %>" data-hook="images_row" class="<%= cycle('odd', 'even')%>">
```

Comment:
As we add more long lived pages `cycle('odd', 'even')` is really gonna show it's limitations, as new uploads don't get striped.